coco_keyprovider/src/grpc/mod.rs

```diff
@@ -0,0 +1,155 @@
+// Copyright (c) 2021 Alibaba Cloud
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+use crate::enc_mods;
+use anyhow::*;
+use jwt_simple::prelude::Ed25519KeyPair;
+use log::*;
+use reqwest::Url;
+use std::net::SocketAddr;
+use std::path::PathBuf;
+use tokio::fs;
+use tonic::{transport::Server, Request, Response, Status};
+
+use key_provider::key_provider_service_server::{KeyProviderService, KeyProviderServiceServer};
+use key_provider::{KeyProviderKeyWrapProtocolInput, KeyProviderKeyWrapProtocolOutput};
+use protocol::keyprovider_structs::*;
+
+pub mod protocol;
+pub mod key_provider {
+    tonic::include_proto!("keyprovider");
```

Comment:
We can add following micro here to avoid the CI failing.
```
#![allow(unknown_lints)]
#![allow(clippy::derive_partial_eq_without_eq)]
#![allow(clippy::redundant_async_block)]
```

Reply:
Thanks a lot. Let me try this!